### PR TITLE
feat(guide): Collection of Custom Formats - HDR Formats - Improve description

### PR DESCRIPTION
**Pull request by TRaSH-**
# Pull Request

## Purpose

Improve and update the Custom Formats Collection Description.

## Approach

Improve and update the Custom Formats Collection Description.
Review the existing descriptions

## Open Questions and Pre-Merge TODOs

===[Radarr]===
- [x] Radarr Collection of Custom Formats HDR Formats

===[Sonarr]===
- [x] Sonarr Collection of Custom Formats HDR Formats

## Requirements

- [x] These changes meet the standards for [contributing](https://github.com/TRaSH-Guides/Guides/blob/master/CONTRIBUTING.md).
- [x] I have read the [code of conduct](https://github.com/TRaSH-Guides/Guides/blob/master/.github/CODE_OF_CONDUCT.md).

### CODE REVIEW

File: includes/cf-descriptions/dolby-vision.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+**Dolby Vision**<br>
+
+Dolby Vision is a dynamic HDR format.
+It potentially offers a better visual experience compared to HDR10 for several reasons. Firstly, it employs dynamic metadata, allowing for varying brightness levels within HDR content and enabling filmmakers to have more precise control over the image appearance. HDR10 relies on static metadata, limiting the HDR effect to be determined only on a per-movie or per-show basis.
```
Comment (zakkarry) [inline includes/cf-descriptions/dolby-vision.md:4]:
```suggestion
Dolby Vision is a dynamic HDR format.
It can offer better visual experiences relative to HDR10. The most significant difference is that it uses dynamic metadata which allows varying brightness levels within HDR content which enables filmmakers to have more precise control over the overall image appearance. HDR10 relies on static metadata which limits the HDR effect to a per-movie or per-episode basis.```

File: includes/cf-descriptions/dv-hdr10.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+*This Custom Format will only match on* `DV HDR10`
+
+This provides Dolby Vision with HDR10 fallback.
```
Comment (zakkarry) [inline includes/cf-descriptions/dv-hdr10.md:3]:
```suggestion
This format provides Dolby Vision *with* HDR10 fallback.
```

File: includes/cf-descriptions/dv-hdr10plus.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+*This Custom Format will only match on* `DV HDR10+`
+
+This provides Dolby Vision with HDR10+ fallback, while HDR10+ also has HDR10 fallback..
```
Comment (zakkarry) [inline includes/cf-descriptions/dv-hdr10plus.md:3]:
```suggestion
This format provides Dolby Vision *with* HDR10+ fallback (with HDR10+ having native HDR10 fallback).
```

File: includes/cf-descriptions/dv-hlg.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+*This Custom Format will only match on* `DV HLG`
+
+**DV HLG**<br>
+This is a very rare HDR Format.
```
Comment (zakkarry) [inline includes/cf-descriptions/dv-hlg.md:4]:
```suggestion
This is a very rare HDR Format that was created by NHK (JP) and the BBC for cable, satellite, and over-the-air TV broadcasts.
```

File: includes/cf-descriptions/dv.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
+*This Custom Format will only match on* `DV` **NOT** *on* `DV HDR10` *etc*.
+
+Depending on the used source (WEB-DL, Bluray or Remux) it will match on the following
+
+You will find this mainly with:
```
Comment (zakkarry) [inline includes/cf-descriptions/dv.md:5]:
```suggestion
Depending on the source (WEB-DL, Bluray, or Remux) this format will match the following:
```

File: includes/cf-descriptions/hdr-formats-info.md [hunk 6]
```diff
@@ -0,0 +1 @@
+💡 As with all HDR formats, Any form of HDR will look great at least better than SDR, but it may vary depending on your equipment and how the studio/filmmaker utilize it.
```
Comment (zakkarry) [inline includes/cf-descriptions/hdr-formats-info.md:1]:
```suggestion
💡 Any form of HDR will look significantly better than SDR on supporting equipment. The overall result will ultimately end up varying, though, based on your hardware and how the studio and filmmaker implemented HDR.
```

File: includes/cf-descriptions/hdr-undefined.md [hunk 7]
```diff
@@ -0,0 +1,5 @@
+Some release groups don't add HDR to their 4K release name
+
+For now it's only FraMeSToR/HQMUX that doesn't add HDR to their release name but in the feature we can add more to it if needed.
```
Comment (zakkarry) [inline includes/cf-descriptions/hdr-undefined.md:3]:
```suggestion
Some release groups don't add HDR to their 4K release naming scheme.

Currently, only FraMeSToR and HQMUX do not include the 'HDR' tag. In the future, we can add more when it's needed.
```

File: includes/cf-descriptions/hdr10.md [hunk 8]
```diff
@@ -0,0 +1,5 @@
+**HDR10**<br>
+
+HDR10 is the most basic HDR format used in consumer media and is supported by all HDR-supported devices.
+
+HDR10 relies on static metadata, limiting the HDR effect to be determined only on a per-movie or per-show basis. Because HDR10 lacks dynamic metadata, it doesn't inform the display exactly how the content should be adjusted. The interaction between display capabilities, video metadata, and the ultimate output (i.e., the presentation of the video) is mediated by the display, resulting in the potential loss of the original producer's intent.
```
Comment (zakkarry) [inline includes/cf-descriptions/hdr10.md:5]:
```suggestion
HDR10 relies on static metadata which limits its effect to a per-movie or per-episode basis. Because of this, it lacks the information for how the content should be adjusted for your specific display. The interaction between display capabilities, video metadata, and the ultimate output (i.e., the presentation of the video) is controlled by the display, which may result in quality that is ultimately not how was originally intended.

File: includes/cf-descriptions/hdr10plus.md [hunk 9]
```diff
@@ -0,0 +1,3 @@
+**HDR10+**<br>
+
+HDR10+ makes this metadata dynamic, allowing it to change for each frame of video. This means every frame is treated to its own set of colors, brightness, and contrast parameters, making for a much more realistic-looking image. Areas of the screen that might have been over saturated under HDR10 will display their full details with HDR10+.
```
Comment (zakkarry) [inline includes/cf-descriptions/hdr10plus.md:3]:
```suggestion
HDR10+ improves upon HDR10 by making its metadata dynamic, allowing for it to change for each frame of video. This means every frame is adjusted to its own set of colors, brightness, and contrast parameters, making for a much more vibrant and realistic-looking image. Areas of the screen that might have been over-saturated with HDR10 will display their full contrast and detail with HDR10+.

File: includes/cf-descriptions/hlg.md [hunk 10]
```diff
@@ -0,0 +1,7 @@
+*This Custom Format will only match on* `DV HLG`
+
+**HLG**<br>
+
+HLG or HLG10 is an HDR format that can be used for both video and still images. The format is backwards compatible with SDR UHD TV but not with older SDR displays that do not implement the Rec. 2020 color standards.
+
+You will find this mainly with Television Movies/Series like National Geographic.
```
Comment (zakkarry) [inline includes/cf-descriptions/hlg.md:7]:
```suggestion
HLG or HLG10 is an HDR format created by NHK (JP) and the BBC that can be used for both video and still images. This format is backward compatible with SDR UHD TV, but not with older SDR displays that do not implement the Rec. 2020 color standards.

You will find this mainly with cable, satellite, and over-the-air TV broadcast series and movies from the likes of BBC and National Geographic.

File: includes/cf-descriptions/pq.md [hunk 11]
```diff
@@ -0,0 +1,5 @@
+**PQ10**<br>
+
+PQ or PQ10 is the most basic HDR format that can be used for both video and still images. It is the same as the HDR10 format without any metadata.
```
Comment (zakkarry) [inline includes/cf-descriptions/pq.md:3]:
```suggestion
PQ (perceptual quantizer) or PQ10 is the most basic HDR format that can be used for both video and still images. It is the same as the HDR10 format without any metadata.

**Review by zakkarry — CHANGES_REQUESTED** [review]
proposed changes, most grammatical but a few additions of information to be complete

also I'm questioning the existence of DV SDR format?

**Comment by yammes08** [comment]
> proposed changes, most grammatical but a few additions of information to be complete
> 
> also I'm questioning the existence of DV SDR format?

DV profile 8.2 is DV with SDR fallback.

**Comment by zakkarry** [comment]
> > proposed changes, most grammatical but a few additions of information to be complete
> > also I'm questioning the existence of DV SDR format?
> 
> DV profile 8.2 is DV with SDR fallback.

Well let me rephrase, i didn't doubt it was something that existed as much as its existence in the context of needing a CF and being released....

poor wording on my part.

**Comment by TRaSH-** [comment]
And to be honest I never have seen a single `DV HLG` or `DV SDR`

File: includes/cf-descriptions/hdr.md [hunk 12]
```diff
@@ -0,0 +1,3 @@
+*This Custom Format will only match on* `HDR` **NOT** *on* `HDR10`, `HDR10+` *etc*
+
+This Custom Format is more of a catch-all for `HDR` scene naming, where we don't know upfront which HDR formats it actually has. It could be `HDR10` or even `PQ`, or whatever.
```
Comment (zakkarry) [inline includes/cf-descriptions/hdr.md:3]:
```suggestion
This Custom Format is a sort of "catch-all" for the `HDR` scene naming scheme where we don't know (based on name alone) which HDR formats are included. It could be `HDR10`, `HDR10+` or even `PQ`.

**Review by zakkarry — CHANGES_REQUESTED** [review]
oops. one more.

**Review by zakkarry — DISMISSED** [review]

**Review by zakkarry — APPROVED** [review]
